app/res/layout/question_widget_text_compact.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+          android:layout_width="0dp"
+          android:layout_weight="1"
```

Comment:
My understanding is that `layout_weight` only does something useful when that view's parent has defined the property `weightSum`. I'm not seeing that anywhere in the PR, so what does this do?

Reply:
If no weightSum is specified, it is computed as the sum of all layout_weight values. We can set it explicitly as well if it's more clear.

Reply:
Hmm but this is the only place that `layout_weight` is being used, so wouldn't that make the effect of this that this view will take up the whole screen? I was assuming the goal was for it to take up half of it, but maybe I'm missing something. (That also makes me realizing that including some screenshots in the PR description would be helpful; would you mind doing that now?)

Reply:
Added some screenshots. So we want the question text to take all of the space available apart from the input edit text plus some padding which is what exactly `layout_weight` 1 is doing in this case.

Reply:
Ok cool, the screenshots help for sure. Looks good to me.